modules/aca/office_booking.rb

```diff
@@ -149,230 +73,78 @@ def on_update
         self[:booking_hide_all] = setting(:booking_hide_all) || false
         self[:hide_all] = setting(:booking_hide_all) || false # for backwards compatibility only
```

Comment:
This is assigned up on line 34 as well.

Reply:
hide_all removed as it has been replaced by booking_hide_all for some time now.